### PR TITLE
PHP CS Fixer - update config

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -85,20 +85,22 @@ return PhpCsFixer\Config::create()
...
-                'property_protected',
-                'property_private',
-                'construct',
-                'destruct',
-                'magic',
-                'phpunit',
-                'method_public',
-                'method_protected',
-                'method_private'
+                'order' => [
```
Comment (keradus) [inline .php_cs.dist]: rootless config is deprecated

**Comment by sebastianbergmann** [comment]
The `5.7` and `6.5` branches need the same change.

**Comment by sebastianbergmann** [comment]
Merged manually, thanks.

**Comment by keradus** [comment]
damn, I didn't noticed it was originally applied on 5.7.
I saw only the commit that was targeting master (gh shows it was applied to master, but not prior brnaches, probably due to way how it was obtained from lower branch)

thanks !

**Comment by sebastianbergmann** [comment]
No worries. Thanks!

**Pull request closed without merging**